src/main/java/org/apache/commons/net/util/Base64.java

```diff
@@ -603,10 +603,10 @@ int avail() {
     }
 
     /**
-     * Decodes a byte[] containing characters in the Base64 alphabet.
+     * Decodes a byte array containing characters in the Base64 alphabet.
      *
      * @param pArray A byte array containing Base64 character data
-     * @return a byte array containing binary data
+     * @return a byte array containing binary data; will return {@code null} if provided byte array is null.
```

Comment:
s/is null/is {@code null}

Reply:
fixed in 7c11889a03a3754a9abd6874cf8b2d06707d0cb9